crates/typed-store/src/metrics.rs

```diff
@@ -1,14 +1,20 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
+use coarsetime::Clock;
 use once_cell::sync::OnceCell;
 use prometheus::{
     exponential_buckets, register_histogram_vec_with_registry,
     register_int_counter_vec_with_registry, register_int_gauge_vec_with_registry, HistogramVec,
     IntCounterVec, IntGaugeVec, Registry,
 };
+use std::cell::RefCell;
 use std::sync::atomic::{AtomicU64, Ordering};
-use std::time::{Duration, UNIX_EPOCH};
-use std::{sync::Arc, time::SystemTime};
+use std::sync::Arc;
+use std::time::Duration;
+
+use rocksdb::{perf::set_perf_stats, PerfContext, PerfMetric, PerfStatsLevel};
+
+thread_local! {static PER_THREAD_ROCKS_PERF_CONTEXT: std::cell::RefCell<PerfContext>  = RefCell::new(PerfContext::default());}
```

Comment:
Would you be able to format this so that it is a bit more readable?
```suggestion
thread_local! {
    static PER_THREAD_ROCKS_PERF_CONTEXT: std::cell::RefCell<PerfContext>  = RefCell::new(PerfContext::default());
}
```